From 3027a8782e4bbee8242e80108bc3fbd376abb5f9 Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Tue, 10 Mar 2020 14:47:08 +0200
Subject: [PATCH] [SW-0] goya2: make: update install IP address

change default IP of installation since host11 is out of order

Change-Id: Id1fda9ed8508badc49446b4628f08bac089671a0
---

diff --git a/scripts/make/goya2/Makefile b/scripts/make/goya2/Makefile
index 2c064a5..9c6ccf6 100644
--- a/scripts/make/goya2/Makefile
+++ b/scripts/make/goya2/Makefile
@@ -22,9 +22,9 @@
 	@$(MAKE) all
 
 .PHONY: goya2-onix-install goya2-onix-install-all
-goya2-onix-install: HL_INSTALL_HOST_IP = emb-srv-host11
-goya2-onix-install-all: HL_INSTALL_HOST_IP = emb-srv-host11
-goya2-onix-install-all: HL_INSTALL_LAB_IP = emb-srv-nuc11
+goya2-onix-install: HL_INSTALL_HOST_IP = emb-srv-host9
+goya2-onix-install-all: HL_INSTALL_HOST_IP = emb-srv-host9
+goya2-onix-install-all: HL_INSTALL_LAB_IP = emb-srv-host9
 goya2-install-host:
 	@$(MAKE) hl-install-host HL_INSTALL_HOST_IP=$(HL_INSTALL_HOST_IP)
 goya2-install-lab:
